### PR TITLE
Matt V's completed homework w01d04 - Events Lab

### PR DESCRIPTION
**Pull request by matthewvilhauer**
Had fun with this one :-)

Only thing I have a question on is how to get the mobile navbar menu button to display the links properly on the mobile site. You can click the button but it does nothing. Not sure what else needs to be added...

### CODE REVIEW

**Comment by nathanallen** [comment]
Looks to me like you're missing `bootstrap.js`. Some of the more "interactive" bootstrap components require javascript. Others just use css.

Glad to see you downloaded bootstrap. Another approach is to use the bootstrap CDN (but then you can't work offline!).

I'm liking what I'm seeing! Good problem solving!

File: keypress/js/app.js [hunk 1]
```diff
@@ -2,4 +2,38 @@ console.log("Sanity Check: JS is working!");
 
 $(document).ready(function() {
 
+    var SpaceBarPresses = 0;
```
Comment (nathanallen) [inline keypress/js/app.js:5]: prefer camelcase: `spaceBarPresses`, or snakecase: `space_bar_presses`

File: change/js/app.js [hunk 2]
```diff
@@ -2,6 +2,49 @@ console.log("Sanity Check: JS is working!");
...
+        return parseInt($("#total").val());
+    };
+    var leftValue = function() {
+        return parseInt($("#left").val());
+    };
+    var rightValue = function() {
+        return parseInt($("#right").val());
+    };
+    console.log(typeof(totalValue()));
+
```
Comment (Nick-Brennan) [inline change/js/app.js:23]: Good stuff, but please define your functions using this format/syntax:

function totalValue(){
     return parseInt($("#total").val());
}
**comments is not displaying the 2 space indent

File: change/js/app.js [hunk 3]
```diff
@@ -2,6 +2,49 @@ console.log("Sanity Check: JS is working!");
...
+        return calculateTotal();
+    });
+
+    $(".reset").on("click", function() {
+        initialize();
+    });
 
 });
+
+
```
Comment (Nick-Brennan) [inline change/js/app.js:43]: Try using the 'change' event

**Pull request closed without merging**